packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -1,7 +1,12 @@
 import 'whatwg-fetch';
+import fetchRetry from 'fetch-retry';
 import { CallApi } from '../types';
 import { CACHE_AVAILABLE, CACHE_KEY, HTTP_STATUS_NOT_MODIFIED, HTTP_STATUS_OK } from '../constants';
 
+const RETRY_COUNT = 3;
```

Comment:
since this is a pretty big change, I wonder if we should make this `2` by default for a _total_ of 3 requests? not really sure what's standard.

Reply:
3 is the default for the package and is a standard i've seen before. if we make this configurable i think it's fine to default it though